### PR TITLE
Design form user

### PR DESCRIPTION
**Pull request by samnangcattor**

### CODE REVIEW

File: app/helpers/supervisor/users_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module Supervisor::UsersHelper
+end
```
Comment (RathanakSreang) [inline app/helpers/supervisor/users_helper.rb]: Remove

**Comment by HorNorin** [comment]
look good

File: app/controllers/supervisor/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+   end
+  end
+
+  def show
+  end
+
+  private
+  def user_params
+    params.require(:user).permit :name, 
+                                 :email,
```
Comment (chuanhtuan) [inline app/controllers/supervisor/users_controller.rb]: bad indent :(

File: config/routes.rb [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
-  get    "login"   => "sessions#new"
-  post   "login"   => "sessions#create"
-  delete "logout"  => "sessions#destroy"
+  root  "static_pages#home"
+  get  "login"  => "sessions#new"
+  post  "login"  => "sessions#create"
+  delete  "logout"  => "sessions#destroy"
   resources :users
+  namespace :supervisor do
+  	resources :users
```
Comment (chuanhtuan) [inline config/routes.rb]: indent 2

File: app/controllers/supervisor/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+class Supervisor::UsersController < ApplicationController
+  def new
+   @user = User.new
+  end
+  
+  def create
+   @user = User.new user_params
+   if @user.save
+     flash[:info] = "Create Successfull"
```
Comment (chuanhtuan) [inline app/controllers/supervisor/users_controller.rb]: i18n

**Pull request merged**